packages/grafana-data/src/dataframe/ArrowDataFrame.ts

```diff
@@ -162,15 +163,18 @@ export function grafanaDataFrameToArrowTable(data: DataFrame): Table {
 }
 
 export function resultsToDataFrames(rsp: any): DataFrame[] {
```

Comment:
Does it make sense to type the `rsp` here if we are making assertions about the argument structure anyway in the body?